docs/managing-minishift.md

```diff
@@ -181,8 +183,20 @@ These are not configurable at the moment and are different for each driver and t
 <a name="mounting-custom-shared-folders"></a>
 ### Mounting custom shared folders
 
-The Boot2Docker and the CentOS image include `cifs-utils`, which allows you to mount CIFS-based shared
```

Comment:
There is a broken link in the section `mounted-host-folders` _Note_ section. Could you also fix it as part of this PR @hferentschik ?